web-services/common/src/main/java/datawave/webservice/common/remote/RemoteHttpService.java

```diff
@@ -570,30 +575,61 @@ public void setConfig(RemoteHttpServiceConfiguration config) {
         this.config = config;
     }
 
+    /**
+     * classes that are instances of IOException that will cause DatawaveRetryHandler to retry with delay subclasses of RemoteHttpService should override this
+     * method if necessary
+     */
+    protected List<Class<? extends IOException>> getUnavailableRetryClasses() {
```

Comment:
Would something like `getRetriableClasses()` be a more succinct and/or future-proof name?

Reply:
All IOExceptions are retriable except for those in nonRetriableClasses

Apache HttpClient uses a delay for response codes that indicate that the service is unavailable:SC_SERVICE_UNAVAILABLE.  Datawave adds SC_NOT_FOUND

Datawave uses the same paradigm in the IOException handler to introduce a delay for IOExceptions that indicate that the service is unavailable.  Previously ConnectException.  This PR adds HostUnknownException.

So it's not just that these are retriable.  They are classes that indicate that the service is unavailable and should be retried with a delay.